app/facilities/config.go

```diff
@@ -1,22 +1,30 @@
 package facilities
 
-import "os"
+import (
+	"os"
+)
 
 type Config struct {
 	DB DBConfig
 }
 
-func NewConfig() Config {
-	conf := Config{
+func NewConfig() (*Config, error) {
+	var port string
+
+	port, ok := os.LookupEnv("db_port")
```

Comment:
This is a suggestion, not mandatory to do. 
This is how I'd write this.

```
port := defaultPostgresPort
if p, ok := os.LookupEnv("db_port"); ok {
    port = p
}
```

Reply:
Looks more Go-ish, I guess)